Makefile

```diff
@@ -151,20 +150,6 @@ check-docs: $(EMBEDMD) $(LICHE) build
 	@$(LICHE) --recursive docs --exclude "cloud.tencent.com" --document-root .
 	@$(LICHE) --exclude "cloud.tencent.com" --document-root . *.md
 
-# errcheck performs static analysis and returns error if any of the errors is not checked.
-.PHONY: errcheck
-errcheck: $(ERRCHECK)
-	@echo ">> errchecking the code"
-	$(ERRCHECK) -verbose -exclude .errcheck_excludes.txt ./cmd/... ./pkg/... ./test/...
-
-# format formats the code (including imports format).
-# NOTE: format requires deps to not remove imports that are used, just not resolved.
-# This is not encoded, because it is often used in IDE onSave logic.
-.PHONY: format
```

Comment:
This is bit different... Why we are removing this? It's nice to have a makefile for formating a file (this also allows format single file)

We use it in Goland as well (:

Reply:
It's part of the `golangci-lint` as well I think, need to double check

Reply:
Brought it back but now it uses `golint-ci`